griptape/drivers/file_manager/base_file_manager_driver.py

```diff
@@ -17,9 +17,17 @@ class BaseFileManagerDriver(ABC):
         loaders: Dictionary of file extension specific loaders to use for loading file contents into artifacts.
     """
 
-    workdir: str = field(kw_only=True)
+    _workdir: str = field(kw_only=True, alias="workdir")
     encoding: Optional[str] = field(default=None, kw_only=True)
 
+    @property
```

Comment:
Should workdir getter/setters be abstract for the base_fmd?

Reply:
Yes great call